db/people.yml

```diff
@@ -2411,9 +2411,9 @@ kknox:
     - begin_year: 2006
       end_year: 2011
       group: bg
-      status: Graduate Researcher
+      status: phd
       advisor: sbillinge
       organization: Columbia University
-      position: phd
+      position: Graduate Researcher
   name: Kevin Knox
-  position: Scientist
+  position: Vice President
```

Comment:
@maak-sdu this is still not passing tests for some reason...did it pass locally?

Reply:
@sbillinge @berrakozer  When I try to test locally, "validate" does is not on the list, when I do regolith -h